packages/java/parser-jvm-core/src/main/java/dev/hilla/parser/core/ScanElementsCollector.java

```diff
@@ -41,7 +42,7 @@ public ScanElementsCollector collect() {
 
         entities = endpoints.stream()
                 .flatMap(cls -> cls.getInheritanceChainStream()
```

Comment:
Should we also parse interfaces implemented by endpoint to find default methods?

Reply:
I'd definitely say so: look at https://github.com/vaadin/hilla/blob/e2f4dc35147235e0ab6ec93ffb25dfbea334d6ee/packages/java/endpoint/src/test/java/dev/hilla/generator/endpoints/superclassmethods/PagedData.java#L27-L28

Reply:
Added